src/util.c

```diff
@@ -267,11 +267,11 @@ void print_test_results(struct ntttcp_test_endpoint *tep)
 		if (write_result_into_xml_file(tep) != 0)
 			PRINT_ERR("Error writing log to xml file");
 	if (tep->test->save_json_log)
-                if (write_result_into_json_file(tep) != 0)
-                        PRINT_ERR("Error writing log to json file");
+		if (write_result_into_json_file(tep) != 0)
+			PRINT_ERR("Error writing log to json file");
 }
 
-size_t execute_system_cmd_by_process(char *command, char *type, char *output)
+size_t execute_system_cmd_by_process(char *command, char *type, char **output)
 {
```

Comment:
why you want to make this change?

Reply:
The variable "os_info" is defined as NULL. If this variable is going to be passed in as an input parameter, we need to pass in its address, otherwise, an error will be reported when running this code.